### ISSUE TITLE
How to generate a simple number table id type

### ISSUE THREAD

**Issue by Kim-Andersen** [kristiandupont/kanel#279]
Hi,
is it possible to generate simple id types like
```ts
export type PostId = number
```
instead of 
```ts
export type PostId = number & { " __flavor"?: 'posts' };
```

I'm trying to use the `kanel` generated types with json schema validators and they don't like `number & { " __flavor"?: 'posts' }` very much.

Thanks in advance.

**Comment by kristiandupont** [comment]
Yes, if you provide the option `makeIdType = innerType => innerType`, you should get that result.

**Comment by Kim-Andersen** [comment]
Ok, thanks a lot.
